src/serialmonitor/usbDetector.ts

```diff
@@ -106,6 +117,20 @@ export class UsbDetector {
         }
     }
 
+    public pauseListening() {
+        if (this._usbDetector) {

```

Comment:
Can we resume the existing stopListening/startListening? The two new APIs seems duplicate